From: Babu Moger <babu.moger@oracle.com>
Date: Sat, 18 Jun 2016 17:40:47 -0700
Subject: ixgbe: Fix minor typo while freeing irq
Patch-mainline: v4.8-rc1
Git-commit: 90c6f87786e65e12cab06e637db8c7ef2f22de95
References: bsc#1024349 FATE#321240 bsc#1024355 FATE#321241

The array subscript increments after the execution of the statement.
So there is no issue here. However it helps to read the code better.

Signed-off-by: Babu Moger <babu.moger@oracle.com>
Tested-by: Andrew Bowers <andrewx.bowers@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/net/ethernet/intel/ixgbe/ixgbe_main.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c
+++ b/drivers/net/ethernet/intel/ixgbe/ixgbe_main.c
@@ -3080,7 +3080,7 @@ static void ixgbe_free_irq(struct ixgbe_
 		free_irq(entry->vector, q_vector);
 	}
 
-	free_irq(adapter->msix_entries[vector++].vector, adapter);
+	free_irq(adapter->msix_entries[vector].vector, adapter);
 }
 
 /**
